### PR TITLE
Made the tPane titles info to render dynamically 

### PR DESCRIPTION
**Pull request by mannycolon**
**This PR adds the following:**
- Added the ability to the tPane fetch data to retrieve gateway language data from the manifest (tS manifest) and save it in the tc-manifest and at the same time in common.
- Made tPane titles render dynamically by getting the language info from the tc-manifest.

---

This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/221)

### CODE REVIEW

**Comment by ihoegen** [comment]
@mannycolon There's an issue with this. manifest.source_translations[0] is undefined when it comes to .usfm file projects.

**Comment by ihoegen** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_[modules/t_pane/TPane.js, line 100 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/221#-KU4j5iw7YF2Dtw1pF0k:-KU4j5ixwbdALubxU_qE:b-j43m2l) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/c97d591930ec7ad714b88f645f27ba15fcb9ec6b/modules/t_pane/TPane.js#L100)):_

> manifest.source_translations[0]
> This proves to be problematic 

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/221)_

**Comment by mannycolon** [comment]
Reviewed 1 of 2 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_[modules/t_pane/TPane.js, line 100 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/221#-KU4j5iw7YF2Dtw1pF0k:-KU5U-ovYLKy3K1Pnhfs:b-ezljzn) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/c97d591930ec7ad714b88f645f27ba15fcb9ec6b/modules/t_pane/TPane.js#L100)):_
<details><summary>

<i>Previously, ihoegen (Ian Hoegen) wrote…</i></summary>



> > manifest.source_translations[0]
> > This proves to be problematic 
> > </details>
> > do you have a .usfm file i can use to fix this??

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/221)_

**Comment by ihoegen** [comment]
Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/221)_

**Comment by ihoegen** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/221#-:-KU9vgPmhvl-3bJ3IjYR:bnfp4nl)_

File: modules/t_pane/TPane.js [hunk 1]
```diff
@@ -91,23 +91,34 @@ class TPane extends React.Component {
...
 
   render() {
+    let targetLanguageName = "";
+    let gatewayLanguageName = "";
+    let gatewayLanguageVersion = "";
+    let manifest = ModuleApi.getDataFromCommon("tcManifest");
+    if (manifest && manifest.target_language){
+      targetLanguageName = manifest.target_language.name;
+    }
+    if (manifest && manifest.source_translations[0]) {
```
Comment (ihoegen) [inline modules/t_pane/TPane.js]: This is still causing errors, drop the [0].

**Pull request merged**